website/docs/tutorials/08-interacting-with-starknet/01-invoke.md

```diff
@@ -0,0 +1,37 @@
+# Invoking contracts
+
+## Overview
+This command allows you to send an invoke transaction with `@external` function entrypoint.
+
+The basic inputs that you need for the commands are:
+- [Signing credentials](#signing)
+- Contract address
+- Function name
+- Inputs to the function
+- Fee specification (concrete amount, or auto-estimation)
+- Network you want to target (i.e. its name or gateway URL)
+
+For detailed API description, see [invoke command reference](../../cli-reference.md#invoke).
+
+## Signing
+Credentials for paying the fee are needed, which are: 
+- Account address (which pays the fee)
+- Private key for that account (from ArgentX, Braavos etc.) - in hex string (i.e. 0xa1c359ef) form. This can be provided with `PROTOSTAR_ACCOUNT_PRIVATE_KEY` env variable or with a file on local filesystem containing that key in plaintext, in which case you should use `--private-key-path`.
+
+Custom signing logic is made possible by using custom signers - see details [here](../06-deploying/01-cli.md/#2-using-a-custom-signer-class).
+
+## Usage example
+
+```shell 
+PROTOSTAR_ACCOUNT_PRIVATE_KEY=0x<redacted> protostar invoke --contract-address 0x4a739ab73aa3cac01f9da5d55f49fb67baee4919224454a2e3f85b16462a911 --function "setter_tester_success" --network testnet --account-address 0x0481Eed2e02b1ff19Fd32429801f28a59DEa630d81189E39c80F2F60139b381a --max-fee auto --inputs 3
```

Comment:
I am not sure if providing a private key directly in the console should be recommended. The key is going to be stored in bash history. 

Why can't Protostar accept private key as an argument?

Reply:
The safest way IMHO is storing this as an env variable, and we are allowing a file stored pkey as well - which is accepted as an argument for this command

Reply:
I used a pkey arg for this example